benchexec/runexecutor.py

```diff
@@ -869,6 +881,8 @@ def postParent(preParent_result, exit_code, base_path):
             if exit_code.value not in [0, 1]:
                 _get_debug_output_after_crash(output_filename, base_path)
 
+            self.parent_cleanup_fn(parent_setup, exit_code, base_path)
```

Comment:
This does not conform to the documented contract of `parent_cleanup_fn`. Also, passing `exit_code` seems fine, but I would hesitate to pass `base_path`.